src/main/java/dansplugins/factionsystem/integrators/DynmapIntegrator.java

```diff
@@ -404,7 +401,7 @@ else if ((ourblks != null) && (node.getWorldName().equalsIgnoreCase(curworld)) &
                 if (m == null) {
                     m = markerSet.createAreaMarker(polyid, name, false, curworld, x, z, false);
                     if (m == null) {
-                        System.out.println(String.format(localeService.get("ErrorAddingAreaMarker"), polyid));
+                        System.out.printf((localeService.get("ErrorAddingAreaMarker")) + "%n", polyid);
```

Comment:
Appreciate that the original code wasn't yours but since we're changing code in the area, perhaps we could use the plugin's logger rather than print statements?

Reply:
I dont think i changed here, i think it automatic changed by reformat code